src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -493,8 +493,9 @@ static int do_map()
     }
 
     if (forker.is_parent()) {
+      global_init_postfork_start(g_ceph_context);
```

Comment:
i tested the proposed using `rbd-nbd map rbd`, it does not crash. maybe i am missing something?